pages/common/vagrant.md

```diff
@@ -0,0 +1,19 @@
+# vagrant
+
+> Configure lightweight, reproducible, and portable development environments
+
+- Create Vagrantfile in current folder with the given Vagrant box
+
+`vagrant init hashicorp/precise32`
```

Comment:
If that is the default Vagrant machine, then by all means we should use it :) but we should mention in the description that it is the default (I thought it was a random example you chose).